pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -152,8 +152,8 @@ let
       fi
       if [[ -L $i ]]; then
         symlinks+=(--symlink "$(${coreutils}/bin/readlink "$i")" "$i")
-      else
-        ro_mounts+=(--ro-bind-try "$i" "$i")
+      elif [[ -e $i ]]; then
+        symlinks+=(--symlink "/.host-etc/''${i#/etc/}" "$i")
```

Comment:
I'm not sure how I feel about this. On one hand, this is additional state inside FHSEnvs that can get out of date with the host but OTOH the previous solution probably had bigger issues than that?

Another thing I'm worrying about is that nested containers such as pressuve-vessel might only pass through certain paths under `/` such as `/etc/` but not `/.host-etc/`; breaking all symlinks.

Reply:
Yeah, the previous solution had essentially zero capacity to adapt to changes in the host `/etc`. Theoretically if files in `/etc` were modified in place, it would carry through before, but activation never modified files in `/etc` in place anyway for atomicity reasons.

As for pressure-vessel, I would think that would show pretty clearly in testing, for example with any game that accesses the internet, since it would lose `resolv.conf`. I tried refreshing the community map list in a proton game, and it seemed to access the internet ok, so I don't think that's an issue.

Reply:
I believe that you tested pressure-vessel, I'm just worrying about situations similar to it. Should have been clearer on the wording.

Perhaps this should be opt-out.

Reply:
I'm a little hesitant to make everything under the sun configurable. And I have trouble imagining a scenario in which a nested container tech has a problem with this that it wouldn't already have had because it didn't know to pass through `/nix`.